sdk/storage/Azure.Storage.Blobs/src/Models/CopyFromUriOperation.cs

```diff
@@ -17,7 +15,7 @@ namespace Azure.Storage.Blobs.Models
     /// request.  Its <see cref="Operation{Int64}.Value"/> upon succesful
     /// completion will be the number of bytes copied.
     /// </summary>
-    internal class CopyFromUriOperation : Operation<long>
+    public class CopyFromUriOperation : Operation<long>
```

Comment:
Should we update the signatures of the `StartCopyFromUri` methods to return this type?  (Just search `Operation<long>` to find them all.)

Reply:
Yes, my bad